src/server/game/Globals/ObjectMgr.h

```diff
@@ -1490,6 +1490,7 @@ class TC_GAME_API ObjectMgr
         bool DeleteGameTele(std::string_view name);
 
         Trainer::Trainer const* GetTrainer(uint32 creatureId) const;
+        auto GetClassTrainers(uint8 classId) const { return Trinity::Containers::MapEqualRange(_classTrainers, classId); }
```

Comment:
nope

Reply:
Do you really want me to write out `Trinity::IteratorPair<WhateverTheFuckThatIteratorTypeIs>`?

Reply:
`Trinity::IteratorPair<std::unordered_multimap<uint8, Trainer const*>::const_iterator>`